core-plugins/docs/Projection-transform.md

```diff
@@ -68,16 +74,17 @@ converted to an int.
 
 Example
 -------
-This example drops the ``'ts'`` and ``'headers'`` fields. It also changes the type of the ``'cost'``
-field to a double and renames it ``'price'``. 
+This example only keeps the ``'id'`` and ``'cost'`` fields. It also changes the type of the ``'cost'``
+field to a double and renames it ``'price'``.
 
     {
         "name": "Projection",
         "type": "transform",
         "properties": {
-            "drop": "ts,headers",
+            "drop": "",
             "convert": "cost:double",
-            "rename": "cost:price"
+            "rename": "cost:price",
+	    "keep": "id,cost"
```

Comment:
Hmm the indent looks fine when I open it to edit but looks out of place when viewed.